ports/ut/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "ut",
```

```suggestion
  "name": "bext-ut",
```
For libraries from `boost-ext`, we are using the `bext-` prefix: https://github.com/microsoft/vcpkg/pull/16302

---

Applied the feedback in 745ff1f, ed56237. Thanks for the point!